featuretools/entityset/entityset.py

```diff
@@ -291,6 +291,25 @@ def add_relationship(self, relationship):
         self.reset_data_description()
         return self
 
+    def set_secondary_time_index(self, entity, secondary_time_index):
+        for time_index, columns in secondary_time_index.items():
+            if is_instance(entity.df, (dd, ks), 'DataFrame') or entity.df.empty:
+                time_to_check = vtypes.DEFAULT_DTYPE_VALUES[entity[time_index]._default_pandas_dtype]
```

Comment:
line length is 102, thoughts on breaking this into two lines?

Reply:
Good idea. Updated.